docs/source/squirrel/functions.rst

```diff
@@ -69,13 +69,16 @@ If you want to pass a function as a parameter to another function, for example a
     printt( req + opt )
   }
 
-  void function CallLiteral( functionref( int, int ) literal )
+  void function CallLiteral( void functionref( int, int ) literal )
   {
     literal( RandomInt( 5 ) )
   }
 
   CallLiteral( FnLiteral )
 
+
+You are able to leave out the return type of the functionref if your file is ``untyped``, however this is discouraged to keep compatabliliy.
```

Comment:
```suggestion
..note ::

  In ``untyped`` files you may leave out the return type of the function reference. Like in function declarations this will default to ``var`` being the return type.
```